src/config-file-schema.test.ts

```diff
@@ -3,14 +3,14 @@ import * as tempy from 'tempy';
 
 import { ConfigFileSchema } from './config-file-schema';
 import { join } from 'path';
-import { JSONSchemaType } from 'ajv';
+import Ajv from 'ajv';
 
 interface TestSchema {
   foo: string;
   baz?: string;
 }
 
-const schema: JSONSchemaType<TestSchema> = {
+const validateSchema = {
```

Comment:
I don't really understand this change. The basic workflow in ajv docs(https://ajv.js.org/guide/typescript.html#utility-types-for-schemas) is to keep this line the way it was and generate the validation function like this:
```
const validate = ajv.compile(schema)
```

Is there a benefit to the way it's been changed here?

Reply:
You're right! I removed the type from the schema definition because the types weren't exactly matching at first, and then I didn't add it back. Fixing now! :)